tests/token.spec.coffee

```diff
@@ -121,6 +121,14 @@ describe 'Request (token):', ->
 							authorizationHeader = response.request.headers.Authorization
 							m.chai.expect(authorizationHeader).to.equal("Bearer #{janeDoeFixture.token}")
 
+					it 'should refresh the token when requested', ->
```

Comment:
NIT: I'd like to see this broken out into it's own describe block, as its a seperate method, something like
```
describe '.refreshToken()', ->
    describe 'given a working /whoami endpoint', ->
        it 'should refresh the token', ->
            # test code
```